python/nav/django/utils.py

```diff
@@ -56,6 +57,39 @@ def get_verbose_name(model, lookup):
     raise FieldDoesNotExist
 
 
+def pp_request(request: HttpRequest, function, *attributes) -> None:
```

Comment:
While this adds some very nice functionality, this name sucks, IMNSHO.

I take it `pp` is meant to convey the concept of `pretty-print` (but it sounds awfully childish when pronounced).  Since it doesn't actually "print" anything, at best this would be cognate with the function `pprint.pformat()`, so how about `pformat_request()` instead?